internal/cmdlineopt.h

```diff
@@ -30,7 +30,7 @@ typedef struct ruby_cmdline_options {
 
     const char *crash_report;
 
-    signed int sflag: 2;
+    signed int sflag: 3;
```

Comment:
2bits will be enough for -1..2.

Reply:
Aren't the bounds of a signed 2 bit integer `-2` thru `1`?
- `00` is `0`
- `01` is `1`
- `10` is `-2`
- `11` is `-1`

Granted, I could rework the code to use these bounds instead, or make it unsigned

Reply:
Ah, yes, you're right.